internal/controllers/app_controller.go

```diff
@@ -222,6 +251,7 @@ func (r *AppReconciler) deleteChart(ctx context.Context, appName string) error {
 }
 
 func (r *AppReconciler) SetupWithManager(mgr ctrl.Manager) error {
+	r.Now = time.Now
```

Comment:
can we move this initialization to
 
```golang
	if err = (&controllers.AppReconciler{
		TemplateReader: storage,
		Client:         mgr.GetClient(),
		Log:            ctrl.Log.WithName("controllers").WithName("App"),
		Scheme:         mgr.GetScheme(),
		HelmFactoryFn: func(namespace string) (controllers.Helm, error) {
			return chart.NewHelmClient(namespace)
		},
	}).SetupWithManager(mgr); err != nil {
		setupLog.Error(err, "unable to create controller", "controller", "App")
		os.Exit(1)
	}

```
asking because of two reasons:
1. SetupWithManager doesn't change AppReconciler (until now)
2. let's do initialization in one place
2.